### PR TITLE
Change registration url to /register

### PR DESCRIPTION
**Pull request by gabrieltron**

### CODE REVIEW

File: docs/quickstart.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ import TabItem from '@theme/TabItem';
 ### Leaf Quickstart
 
 This quickstart tutorial is written help you quickly experience and start building with Leaf.
-First, please [register on our website](https://leafagriculture.com.br/registration/) to receive your credentials.
+First, please [register on our website](https://leafagriculture.com.br/register/) to receive your credentials.
```
Comment (CarlosSam) [inline docs/quickstart.md:12]:
```suggestion
First, please [register on our website](https://withleaf.io/register/) to receive your credentials.
```
the url you wrote does not exists

File: docs/authentication.md [hunk 2]
```diff
@@ -82,5 +82,5 @@ json. If you want it to last only 24h, feel free to set "rememberMe" to "false".
 _After expiration, just make the same request to the same endpoint and you will
 get a new access token. _
 
-[register]: https://leafagriculture.com.br/registration/
+[register]: https://leafagriculture.com.br/register/
```
Comment (CarlosSam) [inline docs/authentication.md:85]:
```suggestion
[register]: https://withleaf.io/register/
```
the url you wrote does not exists

File: docusaurus.config.js [hunk 3]
```diff
@@ -55,7 +55,7 @@ module.exports = {
         // { to: 'docs/', activeBasePath: './', label: '             ', position: 'left'},
         // end workaround to center navigation items
         // {
-        //   to: 'https://leafagriculture.com.br/registration/',
+        //   to: 'https://leafagriculture.com.br/register/',
```
Comment (CarlosSam) [inline docusaurus.config.js:58]: why not use the withleaf.io/register in this one too?

**Review by CarlosSam — CHANGES_REQUESTED** [review]

File: docs/quickstart.md (hunk 1 above)
Comment (gabrieltron) [reply docs/quickstart.md]: Nice catch. I tested the register button but not the ones in the docs. Fixed it in 8721832

File: docs/authentication.md (hunk 2 above)
Comment (gabrieltron) [reply docs/authentication.md]: Fixed in 8721832

File: docusaurus.config.js (hunk 3 above)
Comment (gabrieltron) [reply docusaurus.config.js]: Fixed in  8721832

**Comment by gabrieltron** [comment]
[[Docs] Update Leaf's URL](https://trello.com/c/NyCa61Nt/1755-docs-update-leafs-url)

**Pull request merged**